instaclustr/data_source_bundle_user_list.go

```diff
@@ -0,0 +1,49 @@
+package instaclustr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceBundleUserList() *schema.Resource {
```

Comment:
The readme will need to be updated for this resource and the data source

Reply:
updated